src/utils.py

```diff
@@ -1943,23 +1946,25 @@ def insert_htmlbox(
     # ----------------------------------------------------------------
     # Find a scaling factor that lets our story fit in
     # ----------------------------------------------------------------
-    if scale is True:
-        scale_max = None
-    else:
+    if scale_low == 1:
         scale_max = 1
+    elif scale_low > 0:
+        scale_max = 1 / scale_low
+    else:
+        scale_max = None
```

Comment:
Can this be `scale_max = None if scale_low == 0 else 1 / scale_low` ?

Reply:
sure